flow/project.py

```diff
@@ -358,23 +485,25 @@ class FlowCondition(object):
     This can be used to build a graph of conditions and operations.
 
     :param callback:
-        A callable with one positional argument (the job).
+        A callable with one or multiple positional arguments (job or jobs).
     :type callback:
         callable
     """
 
     def __init__(self, callback):
         self._callback = callback
 
-    def __call__(self, job):
+    def __call__(self, jobs):
```

Comment:
Are we internally just using the list, because this will fail if more than one positional argument is passed.